main.go

```diff
@@ -86,6 +87,21 @@ func getLogLevel() log.Level {
 	return level
 }
 
+func getLogFilePath() string {
+	logFilePath := os.Getenv("SEMAPHORE_AGENT_LOG_FILE_PATH")
+	if logFilePath == "" {
+		return "/tmp/agent_log"
+	}
+
+	parentDirectory := path.Dir(logFilePath)
+	err := os.MkdirAll(parentDirectory, 0644)
```

Comment:
Just for me to know: does this create a directory structure or just one directory?

Reply:
a directory structure: https://pkg.go.dev/os#MkdirAll